pkg/api/v1beta3/types.go

```diff
@@ -200,26 +200,26 @@ type VolumeSource struct {
 	// to see the host machine. Most containers will NOT need this.
```

Comment:
Also, everywhere, except PodTemplateSpec and Binding, ObjectMeta should not be tagged with omitempty.